h2o-core/src/test/java/water/H2OTest.java

```diff
@@ -33,5 +35,18 @@ private void checkObjectId(String id, int seq) {
     assertTrue(id.startsWith("test-desc_test-type"));
     assertTrue(id.endsWith("_" + seq));
   }
+
+  @Test
+  public void testPropertiesMatching() throws Exception {
+      // Any string with a dot after `ai.h2o.` must fail
+      assertFalse(H2O.IGNORED_PROPERTIES.matcher("noprefixnodots").matches());
+      assertFalse(H2O.IGNORED_PROPERTIES.matcher("no.prefix.ever").matches());
+      assertFalse(H2O.IGNORED_PROPERTIES.matcher("ai.h2o.anythingwithdotwillfail.").matches());
+      assertFalse(H2O.IGNORED_PROPERTIES.matcher("ai.ai.h2o.org.eclipse.jetty.LEVEL").matches());
+      assertFalse(H2O.IGNORED_PROPERTIES.matcher("ai.h2o.org.eclipse.jetty.util.log.class").matches());
+      assertFalse(H2O.IGNORED_PROPERTIES.matcher("ai.h2o.org.eclipse.jetty.util.log.StdErrLog").matches());
+      
+      assertTrue(H2O.IGNORED_PROPERTIES.matcher("ai.h2o.hdfs_config").matches());
```

Comment:
it also matches

"blabla.ai.h2o.something" and "ai.h2o."

Do we want to allow the prefix there?